src/components/Background.tsx

```diff
@@ -0,0 +1,5 @@
+const Background = () => {
+  return <div className="chaotic-background"></div>;
+};
+
+export default Background;
```

Comment:
Don't use only named exports where possible: https://dev.to/phuocng/avoid-using-default-exports-a1c
```suggestion
export const Background = () => {
  return <div className="chaotic-background"></div>;
};
```